src/Domain/Structure/InterfaceDataPolicy.hpp

```diff
@@ -0,0 +1,72 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstdint>
+#include <iosfwd>
+
+namespace domain {
+/// \brief Label for a neighboring Element (or Block) that determines how
+/// information is exchanged between neighboring Elements
+///
+/// \details The specific label is determined by the relationship between the
+/// block logical coordinates of the neighboring Elements (Blocks) for the
+/// points on the interface between them.  In two cases there is a simple
+/// relationship between the coordinates:
+/// - CopyProject: in this case the block logical coordinates are identical.
```

Comment:
This is a bit misleading, please clarify that the block logical coordinate _frame_ is the same, or something like that, but the two sides can still have different collocation points (different meshes).